prime-router/docs/features/0005-managing-translation-schemas.md

```diff
@@ -0,0 +1,765 @@
+    </details>
+11. The azure function for validation runs successfully
+    <details>
+      <summary>Local</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-03-29T23:53:00.488Z.txt
+                    previous-valid-2023-02-25T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Staging</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-02T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Production</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-03T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+
+### Validation
+
+The initial validation will mimic how the current integration tests work.  Each translation schema will:
+
+- be checked to make sure it can be successfully
+- ran with its sample input and checked against its sample output
+  - failing translation schemas will be indicated in the output
+
+Some follow-up improvements could include indicating which lines did not match between the sample output and produced value
+or what line was invalid in the schema.
+
+## Operations
+
+### Rollout
+
+1. Implement the code update to support using the URI scheme to determine how to retrieve the file
+2. Update all current translation schemas in source code to use the `file:///` scheme
+3. Create the workflow for creating/updating a translation schema used directly in the convert or translate step
+    - This would be a translation schema that is referenced in a `setting`
+4. Migrate all translation schemas referenced in a `setting` to azure
+5. Evaluate whether schemas used across organizations should be migrated to azure
+6. Evaluate if the `fhir_mapping` files for HL7->FHIR library should move to azure
```

Comment:
Only the end user translations should move?

Reply:
I kept this open ended since it was unclear to me what is an "end user" is going to be... I think that is mostly going to be in the `extension`, but this structure is more limited since we have to conform to the underlying library